shared/api/tokenizer_impl.cc

```diff
@@ -131,6 +133,615 @@ OrtxStatus TokenizerImpl::BatchDecode(const std::vector<span<extTokenId_t const>
   return {};
 }
 
+// Constant string variable to store predefined chat template strings for popular supported models
```

Comment:
can move this big chunk into a new cpp file, like chat_template.cc?

Reply:
done!